intermediate_oop/tic_tac_toe/js/grid.js

```diff
@@ -0,0 +1,59 @@
+function Grid () {
+	this.results = {
+		X: [],
+		O: []
+	}
+	this.move = 0;
+	this.on = true;
+}
+
+Grid.prototype.testWin = function(input) {
+	if (this.results[input].indexOf("s1") !== -1 && this.results[input].indexOf("s2") !== -1 && this.results[input].indexOf("s3") !== -1) {
+		return true;
+	}  else if (this.results[input].indexOf("s4") !== -1 && this.results[input].indexOf("s5") !== -1 && this.results[input].indexOf("s6") !== -1) {
```

Comment:
so many else ifs! This is a good first approach - now that you have it working, can you think of a way to refactor this test to have less code duplication?